lighthouse-core/report/html/report-styles.css

```diff
@@ -1134,7 +1133,6 @@
   text-decoration: none;
   padding: var(--score-container-padding);
 
-  --circle-border-width: 8;
```

Comment:
In order to keep this `circle-stroke-width` var in CSS, we'd need to use both `requestAnimationFrame` and `getComputedStyle` in order to read it. That seemed not worth it. 

(And currently we never change this width)

Reply:
seems like it should be relative to the viewbox specified in the svg anyhow so seems even better 👍